README.md

```diff
@@ -1,6 +1,22 @@
+<div align="center">
+  <img src="./assets/icons/logo.svg" width="320">
+</div>
+
 # Kubeflow Website
 
-[![Netlify Status](https://api.netlify.com/api/v1/badges/80644d22-0685-44d0-83bd-187d55464321/deploy-status)](https://app.netlify.com/sites/competent-brattain-de2d6d/deploys)
+<h4 align="center">
+
+  [![Netlify Status](https://api.netlify.com/api/v1/badges/80644d22-0685-44d0-83bd-187d55464321/deploy-status)](https://app.netlify.com/sites/competent-brattain-de2d6d/deploys)
+  <a href="https://landscape.cncf.io/?item=orchestration-management--scheduling-orchestration--kubeflow">
+    <img src="https://img.shields.io/badge/CNCF%20Landscape-5699C6?logo=cncf&style=social" alt="KubeFlow CNCF Landscape" />
+  </a>
+
+  [![Slack](https://img.shields.io/badge/Slack-4A154B?style=for-the-badge&logo=slack&logoColor=white)](./docs/about/community/#kubeflow-slack-channels)
```

Comment:
This link doesn't work, so maybe it is better to use to full link:
https://www.kubeflow.org/docs/about/community/#kubeflow-slack-channels

Reply:
the slack link is not working.  Agree with @andreyvelich  to use https://www.kubeflow.org/docs/about/community/#kubeflow-slack-channels